docs/02-deployment-guide.md

```diff
@@ -122,6 +124,43 @@ Parameters after the environment ID will be passed to a `cdk destroy` command.
 
 ## Deployment
 
+### Install Prerequisite Software
+
+When running commands inside your EC2 environment, you will need the following software:
```

Comment:
I'm not sure why this is all needed here? We can just assume that you've got a built copy of the system, and you can run the scripts? This guide starts by telling you to go through the getting started instructions, which require you to get a setup where you can deploy an instance of Sleeper anyway. It seems reasonable that if you need to build the system yourself, you need to look at the dev guide?